libra_swarm/src/swarm.rs

```diff
@@ -564,6 +564,25 @@ impl Drop for LibraSwarm {
                 if let LibraSwarmDir::Temporary(temp_dir) = dir {
                     let log_path = temp_dir.path();
                     println!("logs located at {:?}", log_path);
+
+                    // Dump logs for each validator to stdout when `LIBRA_DUMP_LOGS`
+                    // environment variable is set
+                    if env::var_os("LIBRA_DUMP_LOGS").is_some() {
+                        for (peer_id, node) in &self.validator_nodes {
```

Comment:
can we only dump the log for those nodes who terminated early?

Reply:
Done